app/assets/stylesheets/settings.scss

```diff
@@ -1,3 +1,9 @@
 $font-family-bold: "BrownTT Bold";
+$font-family-regular: "BrownTT Regular";
 
-$font-size-36: 2.25rem;
\ No newline at end of file
+$font-size-36: 2.25rem; // 1rem = 16px
```

Comment:
This file can be deleted since it is kind of redundant.
`$font-size-36` should replace, probably, one of the default sizes, maybe $size-3.